draft-ietf-httpbis-message-signatures.md

```diff
@@ -377,8 +377,7 @@ The following sections describe each of these steps in detail.
 5. The signer creates an ordered list of content identifiers representing the message content and signature metadata to be covered by the signature, and assigns this list as the signature's Covered Content.
    * Each identifier MUST be one of those defined in Section 2.
    * This list MUST NOT be empty, as this would result in creating a signature over the empty string.
-   * If the signature's Algorithm name does not start with rsa, hmac, or ecdsa, signers SHOULD include `*created` and `*request-target` in the list.
-   * If the signature's Algorithm starts with rsa, hmac, or ecdsa, signers SHOULD include `date` and `*request-target` in the list.
+   * Signers SHOULD include `*created` and `*request-target` in the list.
```

Comment:
Suggest we make this more generic, especially since it's all advice (SHOULD) and not requirement:

```suggestion
   * Signers SHOULD include `*request-target` in the list.
   * Signers SHOULD include a date stamp, such as the `date` header or the `*created` field in the list.
```